intermittent_failures.md

```diff
@@ -0,0 +1,95 @@
+
+## Policy
+
+With few exceptions, no intermittently failing test or feature is allowed. If you see a failure that appears nondeterministic, immediately remove it following these steps:
+
+1. Determine if the failure is intermittent
+   - If possible, execute on your local machine multiple times to tally a failure rate, and attempt to debug the issue
+   - Execute the Jenkins pipeline as many as 10 times, and save at least one text file with a failing log
+2. The source of intermittent failure should be immediately removed from the pipeline, and tagged unstable
+   - If possible, create a new pull request which reverts or disables the intermittent module, feature, test, or hardware-configuration
+3. [Cut a GitHub Issue](https://github.com/o3de/o3de/issues/new/choose) to track both fixing the underlying issue and re-enabling the test
+   - Add a label for the SIG which maintains that feature 'sig/XYZ' ***OR*** use label 'needs-sig'
+
+### Exceptions
+
+The policy above applies to all code, scripts, and hardware used in the Automated Review Pipeline with the following exceptions:
+
+- It is rarely more risky to remove an unstable critical test (or feature) than to leave it in a known intermittently failing state
+  - If you have a reason to not follow this policy, please reach out to SIG-Testing on [Discord](https://discord.gg/p3padwr58u)!
+- Intermittent failures ocurring on only one Operating System are still intermittent failures, though it is reasonable to conditionally disable
+- Builds or tests which reliably fail at a given commit are **not** considered intermittent
+  - Such failures may appear inconsistent across multiple commits, when thrashed by rapid code changes
+  - While the pipeline makes these difficult to introduce, when they occur it is still appropriate to immediately revert or disable until fixed
+
+## The Sandbox Suite
+
+While fixing a stabilty issue, the test can be onboarded to the Sandbox suite. The intent of this suite is to help collect data on whether a fix has stabilized a test, and allow submitting certain fixes before re-onboarding a test into its suite. This suite runs alongside the Periodic test suite, and does not execute in the Automated Review presubmit pipeline nor in the Branch Update postsubmit runs.
+
+In a sense failures in this pipeline are 'expected' and are only used to collect data, not to prevent a regression of functionality. This data should be informing an in-progress fix, to evaluate whether additional work needs to be done. Tests left active in the Sandbox suite for longer than one month will be disabled to save on infrastructure costs.
+
+For information on changing suite registration, please refer to the O3DE Test Onboarding Guide (document not yet created)
```

Comment:
We should include the information on how to add a test to a Sandbox suite in this document, instead of directing to another document which does not yet exist.

Reply:
I think that other document needs to be delivered, and am working on it currently. I do not want our documentation to become a bloat of different concepts.